public/components/datasources/components/new/new_datasource_card_view.tsx

```diff
@@ -16,33 +16,37 @@ import {
 import React, { useState } from 'react';
 import { NewDatasourceDescription } from './new_datasource_description';
 import s3Svg from '../../icons/s3-logo.svg';
+import prometheusSvg from '../../icons/prometheus-logo.svg';
 import { DatasourceType } from '../../../../../common/types/data_connections';
 
 export interface DatasourceCard {
   name: DatasourceType;
   displayName: string;
   description: string;
   displayIcon: JSX.Element;
+  onClick: () => void;
 }
 
-const Datasources: DatasourceCard[] = [
-  {
-    name: 'OPENSEARCH',
-    displayName: 'OpenSearch',
-    description: 'Connect to self managed OpenSearch clusters',
-    displayIcon: <EuiIcon type="logoOpenSearch" size="xl" />,
-  },
```

Comment:
Removed opensearch for now - if we can add it back later in the right way will do so - this specific card needs to not be shown if datasources is not there